trunner/runners/common.py

```diff
@@ -144,10 +144,21 @@ def __exit__(self, exc_type, exc_val, exc_tb):
     def wait_prompt(self, timeout=8):
         self.plo.expect_exact("(plo)% ", timeout=timeout)
 
-    def cmd(self, cmd, timeout=8):
-        self.plo.send(cmd + '\r\n')
+    def cmd(self, cmd, erase=False, timeout=8):
```

Comment:
Why we are putting the erase logic in a simple function that should only handle sending commands? It's a little bit unintuitive. I'd add a new function called `erase` (similar to `copy` or `go`). I know that `cmd` command do not allow to parse the output, but we don't need to use it.

Reply:
Changes in `PloTalker` have been applied in the last fp as we discussed it.